projects/ion/src/lib/popover/popover.directive.ts

```diff
@@ -135,6 +135,22 @@ export class IonPopoverDirective implements OnDestroy {
     }
   }
 
+  @HostListener('document:click', ['$event'])
+  onDocumentClick(event: MouseEvent): void {
```

Comment:
You'll need to remove this listener from document when the popover closes, otherwise this will be listening to documents clicks indefinitely.

Reply:
It will no longer be necessary because of the directive made in #662